Assets/PatchKit Patcher/Scripts/AppData/FileSystem/RetryStrategy.cs

```diff
@@ -12,8 +12,8 @@ public class RetryStrategy : IRequestRetryStrategy
     {
         private static readonly DebugLogger DebugLogger = new DebugLogger(typeof(RetryStrategy));
 
-        public const int DefaultTryCount = 10;
-        public const int DefaultDelayMsec = 500;
+        public const int DefaultTryCount = 5;
+        public const int DefaultDelayMsec = 10;
```

Comment:
Isn't `RetryStrategy` used somewhere else where this change might break previous behaviour?